editor/filesystem_dock.cpp

```diff
@@ -76,6 +76,7 @@ bool FileSystemDock::_create_tree(TreeItem *p_parent, EditorFileSystemDirectory
 	subdirectory_item->set_metadata(0, lpath);
 	if (!p_select_in_favorites && (path == lpath || ((display_mode == DISPLAY_MODE_SPLIT) && path.get_base_dir() == lpath))) {
 		subdirectory_item->select(0);
+		subdirectory_item->set_as_cursor(0);
```

Comment:
Please add a comment to explain why this line is necessary:

```suggestion
		// Prevent crash when trying to collapse or expand a folder when nothing is selected.
		subdirectory_item->set_as_cursor(0);
```